src/org/omegat/languagetools/LanguageToolWrapper.java

```diff
@@ -118,13 +167,18 @@ protected JLanguageTool getLanguageToolInstance(Language ltLang) {
     @Override
     public synchronized List<Mark> getMarksForEntry(SourceTextEntry ste, String sourceText, String translationText,
             boolean isActive) throws Exception {
+
         if (translationText == null || !isEnabled()) {
```

Comment:
Sorry for giving you bad information in the dev ML. `translationText` is indeed required, so the multiple null checks were redundant. Thanks for catching that.